Source/drlg_l4.cpp

```diff
@@ -1323,6 +1306,7 @@ void GenerateLevel(lvl_entry entry)
 	constexpr int Minarea = 173;
 	int ar;
 	bool doneflag;
+	uint8_t floorID = 6;

```

Comment:
```suggestion
	constexpr uint8_t HellFloorID = 6;
```
This variable is not modified so can be a constexpr, also suggested a more descriptive name to differentiate the different uses of the resulting function.

Reply:
I think this is used in multiple places so maybe it belongs in a more general cleanup